speech/cloud-client/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -104,75 +123,72 @@ public void run() {
       responseObserver =
           new ResponseObserver<StreamingRecognizeResponse>() {
-
             ArrayList<StreamingRecognizeResponse> responses = new ArrayList<>();
 
             public void onStart(StreamController controller) {
               referenceToStreamController = controller;
             }
 
             public void onResponse(StreamingRecognizeResponse response) {
-
               responses.add(response);
-
               StreamingRecognitionResult result = response.getResultsList().get(0);
-
               Duration resultEndTime = result.getResultEndTime();
-
               resultEndTimeInMS = (int) ((resultEndTime.getSeconds() * 1000)
-                                        + (resultEndTime.getNanos() / 1000000));
-
+                      + (resultEndTime.getNanos() / 1000000));
               double correctedTime = resultEndTimeInMS - bridgingOffset
-                                          + (STREAMING_LIMIT * restartCounter);
-              DecimalFormat format = new DecimalFormat("0.#");
-
+                      + (STREAMING_LIMIT * restartCounter);
+              DecimalFormat format = new DecimalFormat();
+              format.setMinimumIntegerDigits(2);
               SpeechRecognitionAlternative alternative = result.getAlternativesList().get(0);
               if (result.getIsFinal()) {
                 System.out.print(GREEN);
                 System.out.print("\033[2K\r");
-                System.out.printf("%s: %s\n", format.format(correctedTime),
-                                                  alternative.getTranscript());
-
+                System.out.printf("%s: %s [confidence: %.2f]\n", convertMillisToDate(correctedTime, format),
+                        alternative.getTranscript(),
+                        alternative.getConfidence()
+                );
                 isFinalEndTime = resultEndTimeInMS;
                 lastTranscriptWasFinal = true;
               } else {
                 System.out.print(RED);
                 System.out.print("\033[2K\r");
                 System.out.printf("%s: %s", format.format(correctedTime),
-                                                  alternative.getTranscript());
-
+                        alternative.getTranscript()
+                );
                 lastTranscriptWasFinal = false;
               }
             }
 
-            public void onComplete() {}
-
-            public void onError(Throwable t) {}
+            public void onComplete() {
+            }
 
+            public void onError(Throwable t) {
+            }
           };
-
       clientStream = client.streamingRecognizeCallable().splitCall(responseObserver);
 
+
       RecognitionConfig recognitionConfig =
-          RecognitionConfig.newBuilder()
-              .setEncoding(RecognitionConfig.AudioEncoding.LINEAR16)
-              .setLanguageCode("en-US")
-              .setSampleRateHertz(16000)
-              .build();
+              RecognitionConfig.newBuilder()
```

Comment:
Hey, why did these lines change? Was an automatic formatter run? Could we remove all whitespace changes from this PR so the diff just shows the behavioral differences? :)